test/resources/controls/aws_parallelcluster_install/nfs_spec.rb

```diff
@@ -0,0 +1,15 @@
+control 'nfs' do
+  title 'NFS resource'
+
+  if virtualization.system != 'docker'
+    # Check nfsd process is running
+    describe command('ps aux') do
+      its('stdout') { should match(/nfsd/) }
+    end
+
+    # Check version of NFS
```

Comment:
Nitpick: if this fails, would it be clear from the logs what's wrong?

Reply:
not so clear, I agree we can improve it